pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -441,14 +445,9 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(
 				bindToPort:     bindToPort,
 			}
 
-			// Update the values here so that the plugins use the right ports
-			// uds values
-			// TODO: all plugins need to be updated to account for the fact that
-			// the port may be 0 but bind may have a UDS value
-			// Inboundroute will be different for
+			instance.Endpoint.Family = model.AddressFamilyTCP
```

Comment:
TCP even for UDS?

Reply:
There is no UDS in the inbound path for now.